core/src/lib/rwlock.cc

```diff
@@ -176,7 +176,7 @@ int RwlReadunlock(brwlock_t* rwl)
  * Lock for write access, wait until locked (or error).
  *   Multiple nested write locking is permitted.
  */
-int RwlWritelock_p(brwlock_t* rwl, const char*, int)
```

Comment:
Should we print a Dmsg with a low priority with those parameters instead of deleting them ?
That information can come in handy when trying to debug lock related issues.